python/pyspark/ml/feature.py

```diff
@@ -1014,10 +1015,21 @@ def setOutputCol(self, value):
         """
         return self._set(outputCol=value)
 
+    def createTransformFunc(self):
+        self._transfer_params_to_java()
+        return self._java_obj.createTransformFunc()
+
+    def outputDataType(self):
+        self._transfer_params_to_java()
+        return self._java_obj.outputDataType()
+
+    def validateInputType(self, inputType):
+        self._transfer_params_to_java()
+        self._java_obj.validateInputType(inputType)
+
```

Comment:
Do we still need to support Python 2.7? I have to implement the above three methods to make Python 2.7 pass. I tried to put _ in front of these method names, but 2.7 failed with ```TypeError: Can't instantiate abstract class DCT with abstract methods createTransformFunc, outputDataType, validateInputType```

Reply:
For now, yeah we need to support Python 2, still